### ISSUE TITLE
Some images are missing og:description

### ISSUE THREAD

**Issue by kauffj** [lbryio/spee.ch#253]
https://spee.ch/f/jeremy-toy-factory.gif

has

`<meta property="og:description" content="undefined">`

**Comment by Smurphonomics** [comment]
I've looked at the code and I think the issue is **missing spaces** at the end of `showUrl` and `title` . These seem to be the only two **og tags** `undefined`.

I've not set up the app to check, but I remember having a similar issue a few years back.

**Code found at:**

https://github.com/lbryio/spee.ch/blob/master/helpers/handlebarsHelpers.js

```
<meta property="og:title" content="${title}"> 
<meta property="og:url" content="${showUrl}" > 
<meta property="og:site_name" content="Spee.ch" > 
<meta property="og:description" content="${description}">
```
**Should be:**
```
<meta property="og:title" content="${title}"> 
<meta property="og:url" content="${showUrl}"> 
<meta property="og:site_name" content="Spee.ch"> 
<meta property="og:description" content="${description}">
```
If @billbitt can check this, I'll submit a PR.

Also... There seems to be some issues with gif controllers. Might be good to remove these completely.

Might be better to centre images like we have the videos for better appearance.

https://github.com/lbryio/spee.ch/pull/250/commits/7ba5b09a237f65aeebf93742160350a2bb9d05c4

What do you think?

**Comment by billbitt** [comment]
@Smurphonomics I will make the spacing change, but my guess is the root of this problem is that some assets are published without a description, so a default description needs to be inserted for the meta tag when this is the case.

**Comment by billbitt** [comment]
@Smurphonomics can you DM me on discord or create a separate issue for the gif controller issues?

**Comment by billbitt** [comment]
merged fix into master via #261
